client/src/main/java/io/paradoxical/cassieq/api/client/CassandraQueueApi.java

```diff
@@ -40,11 +43,17 @@ static CassandraQueueApi createClient(String baseUri) {
     @GET("api/v1/queues/{queueName}/messages/next")
     Call<GetMessageResponse> getMessage(
             @Path("queueName") QueueName queueName,
-            @Query("invisibilityTime") Long invisibilityTimeSeconds);
+            @Query("invisibilityTimeSeconds") Long invisibilityTimeSeconds);
 
     @POST("api/v1/queues/{queueName}/messages")
     Call<ResponseBody> addMessage(@Path("queueName") QueueName queueName, @Body Object message);
 
+    @PUT("api/v1/queues/{queueName}/messages")
+    Call<UpdateMessageResponse> updateMessage(
+            @Path("queueName") QueueName queueName,
+            @Query("popReceipt") String popReciept,
```

Comment:
we may want to consider changing the pop receipt to be a part of the resource. but since its a bigger change we should discuss